pkg/venafi/cloud/cloud_test.go

```diff
@@ -86,7 +86,7 @@ func TestGenerateRequest(t *testing.T) {
 
 	keyTypeRSA := certificate.KeyTypeRSA
 	keyTypeEC := certificate.KeyTypeECDSA
-	keyTypeED25519 := certificate.KeyTypeED25519
+	KeyTypeED25519 := certificate.KeyTypeED25519
```

Comment:
@inteon why did you change this to PascalCase?

Reply:
thanks for noting this, this was a search-replace error.